adapters/sweden.js

```diff
@@ -0,0 +1,174 @@
+'use strict';
+
+import { REQUEST_TIMEOUT } from '../lib/constants';
+import { default as baseRequest } from 'request';
+const request = baseRequest.defaults({timeout: REQUEST_TIMEOUT});
+import _ from 'lodash';
+import { default as moment } from 'moment-timezone';
+import cheerio from 'cheerio';
+
+exports.name = 'sweden';
+
+exports.fetchData = function (source, cb) {
+  request(source.url, function (err, res, body) {
+    if (err || res.statusCode !== 200) {
+      return cb({message: 'Failure to load data url.'});
+    }
+
+    // Wrap everything in a try/catch in case something goes wrong
+    try {
+      // Format the data
+      var data = formatData(body);
+
+      // Make sure the data is valid
+      if (data === undefined) {
+        return cb({message: 'Failure to parse data.'});
+      }
+      cb(null, data);
+    } catch (e) {
+      return cb({message: 'Unknown adapter error.'});
+    }
+  });
+};
+
+var formatData = function (result) {
+  var measurements = [];
+
+  var getCoordinates = function (id) {
+    switch (id) {
+      case 'Högdalen':
```

Comment:
Could you make a note in here where these coordinates come from? Ideally we'd be getting them programmatically somehow, but that's not always possible.